tests/test_multiple_files.py

```diff
@@ -0,0 +1,46 @@
+import tempfile
+from unittest import mock
+
+import xarray as xr
+
+from metofficedatahub.multiple_files import MetOfficeDataHub, save_to_zarr
+from tests.conftest import mocked_requests_get
+
+
+def test_init():
+    _ = MetOfficeDataHub(client_id="fake", client_secret="fake")
+
+
+@mock.patch("requests.get", side_effect=mocked_requests_get)
+def test_download_all_none(mock_get):
+    """Check that if there are no order ids, then no data is downloaded"""
+    datahub = MetOfficeDataHub(client_id="fake", client_secret="fake")
+    datahub.download_all_files(order_ids=[])
```

Comment:
shouldn't we assert something here?
eg

```python
data = datahub.download_all_files(...)
assert len(data) == 0
```

or something like that?

Reply:
thanks, yea ill add that